lib/rucio/db/sqla/migrate_repo/versions/7ec22226cdbf_new_replica_state_for_temporary_.py

```diff
@@ -35,11 +35,14 @@ def upgrade():
 
     schema = context.get_context().version_table_schema + '.' if context.get_context().version_table_schema else ''
 
-    if context.get_context().dialect.name in ['oracle', 'postgresql']:
+    if context.get_context().dialect.name == 'oracle':
         drop_constraint('REPLICAS_STATE_CHK', 'replicas', type_='check')
         create_check_constraint(constraint_name='REPLICAS_STATE_CHK', table_name='replicas',
                                 condition="state in ('A', 'U', 'C', 'B', 'D', 'S', 'T')")
 
+    elif context.get_context().dialect.name == 'postgresql':
+        pass  # too complicated on PostgreSQL -- leave the ENUM as it is
```

Comment:
Didn't you already do this above?

https://github.com/rucio/rucio/blob/5cc5020284ea490386fb1f97f53f4f0674abdfec/lib/rucio/db/sqla/migrate_repo/versions/01eaf73ab656_add_new_rule_notification_state_progress.py#L45-L48

Reply:
will check